librosa/core/constantq.py

```diff
@@ -172,46 +176,32 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
 
     # Determine required resampling quality
     Q = float(filter_scale) / (2.0**(1. / bins_per_octave) - 1)
-
     filter_cutoff = fmax_t * (1 + filters.window_bandwidth('hann') / Q)
-
     nyquist = sr / 2.0
-
     if filter_cutoff < audio.BW_FASTEST * nyquist:
         res_type = 'kaiser_fast'
-    elif filter_cutoff < audio.BW_BEST * nyquist:
-        res_type = 'kaiser_best'
     else:
         res_type = 'kaiser_best'
 
-    cqt_resp = []
-
-    len_orig = len(y)
-
     y, sr, hop_length = __early_downsample(y, sr, hop_length,
```

Comment:
So instead of 181 et seq., we test if the upper edge is above some factor relative to our threshold, and if so do some special-case kaiser_best-type stuff, but for the common case we fall through and proceed as usual.